packages/editor-ui/src/components/WorkflowCard.vue

```diff
@@ -169,15 +168,8 @@ export default defineComponent({
 		},
 	},
 	methods: {
-		async onClick(event: Event) {
-			if (
-				this.$refs.cardActions === event.target ||
-				this.$refs.cardActions?.contains(event.target)
-			) {
-				return;
-			}
-
-			if (event.metaKey || event.ctrlKey) {
+		async onClick(event?: KeyboardEvent | PointerEvent) {
+			if (event?.metaKey || event?.ctrlKey) {
```

Comment:
```suggestion
if (
				this.$refs.cardActions === event?.target ||
				this.$refs.cardActions?.contains(event?.target)
			) {
				return;
			}
			if (event.metaKey || event.ctrlKey) {
```

Reply:
@krynble that was checking manually if the `ref="cardActions"` div is the event target or if it contains the event target but it's easier to stop the event on this div by adding `@click.stop`.

The purpose of both solution is to stop opening the workflow when you click on the right side area of the workflow card (where there are other actions)